src/app/GitExtensions.Extensibility/DebugHelpers.cs

```diff
@@ -38,6 +39,22 @@ public static void Fail(string message)
     }
 #pragma warning restore CS8763 // A method marked [DoesNotReturn] should not return.
 
+    [Conditional("DEBUG")]
+    public static void Trace(string message, [CallerMemberName] string caller = "")
+    {
+        const char noBreakSpace = '\u00a0';
```

Comment:
Add a comment that `:{noBreakSpace}` is required to forward the message in debug?